validation/__init__.py

```diff
@@ -504,21 +504,32 @@ class PhoneNumber(Unicode):
     >>> p.to_python('223-456-789o') # testing mixed letters
     Traceback (most recent call last):
     ...
-    ValidationException: Please enter a 10 digit number, with area code, in the form ###-###-####
+    ValidationException: Please enter a 10 digit phone number with optional +country code in the format +#* ###-###-####
     >>> p.to_python('223  456  7890') # testing weird spacing
     u'2234567890'
-    >>> p.to_python('1-223-456-7890') # always throws away leading 1s
-    u'2234567890'
+    >>> p.to_python('+1-223-456-7890') # testing country code with +
+    u'+12234567890'
+    >>> p.to_python('1-223-456-7890') # testing country code missing +
+    u'+12234567890'
     >>> p.to_python('112-456-7890') # area code can't start with 1
     Traceback (most recent call last):
     ...
-    ValidationException: Please enter a 10 digit number, with area code, in the form ###-###-####
-    >>> p.to_python('2-223-456-7890') # throw away country code
-    u'2234567890'
+    ValidationException: Please enter a 10 digit phone number with optional +country code in the format +#* ###-###-####
+
+    >>> p.from_python('+12234567890')
+    '+1 (223) 456-7890'
+    >>> p.from_python('+442234567890')
+    '+44 (223) 456-7890'
+    >>> p.from_python('442234567890') # No plus, ignored
```

Comment:
I couldn't decide, should this only work for the US? If yes, then yeah, it should error on anything other than a leading 1 when there is a country code.